clair/clair.go

```diff
@@ -57,6 +58,7 @@ func (imageAnalysis ImageAnalysis) ShortName(l v1.Layer) string {
 //Config configure Clair from configFile
 func Config() {
 	uri = fmtURI(viper.GetString("clair.uri"), viper.GetInt("clair.port")) + "/v1"
+	host = viper.GetString("clair.host")
```

Comment:
I wasnt sure what the default value should be for this. I think leaving it as a blank string is ok though